airflow/providers/apache/beam/example_dags/example_beam.py

```diff
@@ -309,7 +309,7 @@
 
     wait_for_python_job_dataflow_runner_async_done = DataflowJobStatusSensor(
         task_id="wait-for-python-job-async-done",
-        job_id="{{task_instance.xcom_pull('start_python_job_dataflow_runner_async')['dataflow_job_id']}}",
+        job_id="{{task_instance.xcom_pull('start_python_job_dataflow_runner_async')['id']}}",
```

Comment:
According to the source, `dataflow_job_id` should works. 
https://github.com/apache/airflow/blob/d83e5f20ae1c041a104805f9fbe0643799c31996/airflow/providers/apache/beam/operators/beam.py#L443

Reply:
I agree, based on code of beam.py yes. But ,probably it will better to have a consistency between operators for developers and supportability..

What do you think ?

Reply:
We can think about standardizing it, but we should also try to keep it backwards compatible.
if we want to standardize it, we should look at this problem globally.  could you prepare a comparison of what it looks like in other operators?

Reply:
As we discussed, I removed this change from PR.